docs/user/security/rbac_tutorial.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[space-rbac-tutorial]]
+=== Tutorial:  Use role-based access control to customize Kibana spaces
+
+With role-based access control (RBAC), you can provide users access to data, tools,
+and Kibana spaces.  In this tutorial, you will learn how to configure roles
+that provide the right users with the right access to the data, tools, and
+Kibana spaces.
+
+[float]
+==== Scenario
+
+Our user is a web developer working on a bank's
+online mortgage service.  The web developer has these 
+three requirements:
+
+* Have access to the data for that service 
+* Build visualizations and dashboards
+* Monitor the performance of the system
+
+You'll provide the web developer with the access and privileges to get the job done.
+
+[float]
+==== Prerequisites
+
+To complete this tutorial, you'll need the following:
+
+*  **Administrative privileges**: You must have a role that grants privileges to create a space, role, and user. See the {ref}/built-in-roles.html[built-in] roles.
+*  **A space**: In this tutorial, use `Dev Mortgage` as the space 
+name. See <<spaces-managing, view, create, and delete spaces>> for 
+details on creating a space.
```

Comment:
Would something like `<<spaces-managing, spaces management>>` or `<<spaces-managing, managing spaces>>` work here? My initial impression is that this comes across as a little wordy, but I do see that you're matching the linked section heading.

This is more of a question than a suggestion. I'm not sure what the preferred approach is here.

Reply:
Yes, I think that is a good suggestion.  Changed.